src/test/rustdoc-ui/ignore-block-help.stderr

```diff
@@ -7,11 +7,13 @@ LL | | /// let heart = '❤️';
 LL | | /// ```
    | |_______^
    |
-   = note: error from rustc: character literal may only contain one codepoint
-help: `ignore` code blocks require valid Rust code for syntax highlighting. Mark blocks that do not contain Rust code as text
+   = note: `#[warn(rustdoc::invalid_rust_codeblocks)]` on by default
+help: `ignore` code blocks require valid Rust code for syntax highlighting; mark blocks that do not contain Rust code as text: ```text
+  --> $DIR/ignore-block-help.rs:3:5
    |
-LL | /// ```text,ignore (to-prevent-tidy-error)
-   |     ^^^^^^^^
+LL | /// ```ignore (to-prevent-tidy-error)
+   |     ^^^
+   = note: error from rustc: character literal may only contain one codepoint
```

Comment:
Is there an existing issue about this not pointing to the span that contains the error? Would be nice to fix in the future.

Reply:
Why was this behavior changed in this PR?

Reply:
The `note` just moved below the `help`, it wasn't attached to the correct span before either.

Reply:
The existing issue is https://github.com/rust-lang/rust/issues/67857.

Reply:
> it wasn't attached to the correct span before either.

Ah, I see 👍